unstructured_inference/models/largemodel.py

```diff
@@ -0,0 +1,171 @@
+import os
+from typing import List, Optional
+
+import numpy as np
+import torch
+from PIL.Image import Image
+from transformers import (
+    AutoTokenizer,
+    DonutImageProcessor,
+    DonutProcessor,
+    VisionEncoderDecoderModel,
+)
+
+from unstructured_inference.inference.layoutelement import LocationlessLayoutElement
+from unstructured_inference.models.unstructuredmodel import UnstructuredElementExtractionModel
+
+MODEL_TYPES = {
+    "large_model": {
+        "tokenizer_name": "xlm-roberta-large",
+        "pre_trained_model_name": "unstructuredio/ved-fine-tuning",
+    },
+}
+
+LABEL_MAP = [
+    "Title",
+    "Abstract",
+    "Headline",
+    "Subheadline",
+    "Text",
+    "Table",
+    "List",
+    "List-item",
+    "Page number",
+    "Header",
+    "Footer",
+    "Address",
+    "Author",
+    "Chart",
+    "Caption",
+    "Formula",
+    "Picture",
+    "Advertisement",
+    "Link",
+    "Misc",
+    "Field-Name",
+    "Value",
+    "Threading",
+    "Metadata",
+    "Form",
+]
+
+# NOTE(alan): Order matters with these
```

Comment:
May be worth a comment on how the tags work, since it's not completely obvious (at least not to me)

Reply:
Expanded the note with more detail.